l10n-pocket/en/nav.ftl

```diff
@@ -35,14 +35,17 @@ pocket-mobile-nav-beta = Beta
 
 ### Platform navigation is used in some pages:
 
-### Pocket homepage https://www-dev-pocket.allizom.org
-### Add page URL https://www-dev-pocket.allizom.org/Add
-### All platform pages https://www-dev-pocket.allizom.org/ios
-### https://www-dev-pocket.allizom.org/android
-### https://www-dev-pocket.allizom.org/chrome
-### https://www-dev-pocket.allizom.org/safari
-### https://www-dev-pocket.allizom.org/opera
-### https://www-dev-pocket.allizom.org/edge
+## Pocket homepage https://www-dev-pocket.allizom.org
+## Add page URL https://www-dev-pocket.allizom.org/add
+## All platform pages
+## https://www-dev-pocket.allizom.org/ios
```

Comment:
@alexgibson I just want to double check with you and see if this is what you meant by making all the URLs group level comments? I added `##` for each of the URLs I was referencing. I'm still wrapping my head around Fluent comments and want to make sure we're on the same page.

Reply:
There's a good primer here on comments in the Fluent docs: https://projectfluent.org/fluent/guide/comments.html

The comments in this file (and the other ftl files in this PR) could probably use some clean up:

- Single hash comments (`#`) should be bound only to individual strings / messages (i.e. they should appear on the line directly before a string they are associated with). Right now they are being used throughout this file as group comments to describe groupings of messages. We should probably be using group level comments (`##`) here instead? There are also cases where they aren't bound to any specific string content (e.g. `# pocket/includes/platform-nav.html` at the start of the file?).
- Tripple hash comments (`###`) apply at the file level, so are only typically used at the beginning of a file (e.g. to describe the page URL). Right now we have a second file level comment (`### Platform navigation is used in some pages`) being used as a group level comment.

Reply:
This is good information to write down, thanks for explaining!

Reply:
That's correct 👍